siliconcompiler/tools/openroad/scripts/sc_screenshot.tcl

```diff
@@ -14,7 +14,7 @@ sc_save_image "screenshot" "outputs/${sc_design}.png" $sc_resolution
 
 gui::restore_display_controls
 
-if { [dict exist $sc_cfg tool $sc_tool task $sc_task {var} include_report_images] &&
+if { [dict exists $sc_cfg tool $sc_tool task $sc_task {var} include_report_images] &&
```

Comment:
Turns out this was okay since `dict` will allow partial subcommands, but `dict exists` is technically the full command.